resources/translations/en.edn

```diff
@@ -185,6 +187,8 @@
              :show "Show"
              :show-less "Show less"
              :show-more "Show more"}
+  :change-form {:change-form-intro "You are about to change the form of the following catalogue items. The changes will be made one at a time, so that each old item is terminated and a new copy with the chosen form will be activated."
```

Comment:
This is referred to as :t.administration/change-form-intro in the code, instead of :t.change-form/change-form-intro, as it's defined here.